ibis/util.py

```diff
@@ -15,12 +15,20 @@
 from __future__ import print_function
 
 import types
+from contextlib import contextmanager
 
 import ibis.compat as compat
-
 from ibis.config import options
 
 
+@contextmanager
+def ignoring(*exceptions):
```

Comment:
This exists in the standard library as `contextlib.suppress` since Python 3.4.

How about something in`ibis/compat.py` like:

```python
try:
    from contextlib import suppress
except ImportError:
    import contextlib


    @contextlib.contextmanager
    def suppress(...):
        ...
```

Reply:
Great :) I wasn't aware.